### PR TITLE
Make image cli

### PR DESCRIPTION
**Pull request by Nixolay**
update list creation images in makefile

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -133,21 +135,21 @@ packages: clean build tar rpm deb
...
-	for service in "filter" "notifier" "api" "checker" ; do \
+	for service in "filter" $(services) ; do \
 		docker build --build-arg MoiraVersion=${VERSION_FEATURE} --build-arg GO_VERSION=${GO_VERSION} --build-arg GIT_COMMIT=${GIT_HASH} -f Dockerfile.$$service -t moira/$$service-${MARK_UNSTABLE}:${VERSION_FEATURE} . ; \
 		docker push moira/$$service-${MARK_UNSTABLE}:${VERSION_FEATURE} ; \
 	done
 
 .PHONY: docker_nightly_images
 docker_nightly_images:
-	for service in "filter" "notifier" "api" "checker" ; do \
+	for service in $(services) ; do \
```
Comment (litleleprikon) [inline Makefile]: Filter is forgotten here

File: Makefile [hunk 2]
```diff
@@ -21,6 +21,8 @@ VENDOR := "SKB Kontur"
 URL := "https://github.com/moira-alert/moira"
 LICENSE := "MIT"
 
+services := "notifier" "api" "checker" "cli"
```
Comment (litleleprikon) [inline Makefile]: Let's rename it as `SERVICES` or better `MAIN_TARGETS`

Comment (Nixolay) [reply Makefile]: fix it

File: Makefile (hunk 1 above)
Comment (Nixolay) [reply Makefile]: Corrected

File: Makefile (hunk 2 above)
Comment (Nixolay) [reply Makefile]: Corrected

Comment (Nixolay) [reply Makefile]: corrected

**Pull request merged**